openedx/core/djangoapps/credit/api.py

```diff
@@ -457,6 +470,7 @@ def get_credit_requirement_status(course_key, username, namespace=None, name=Non
             "namespace": requirement.namespace,
             "name": requirement.name,
             "display_name": requirement.display_name,
+            "order": requirement.order,
```

Comment:
We don't need to expose this currently as an ordered list is returned through this API. I just added that so that we might need that in future. But if you say it could be removed.